hydromt/io/writers.py

```diff
@@ -92,32 +53,15 @@ def configwrite(config_fn: Union[str, Path], cfdict: dict, **kwargs) -> None:
         function.
     """
     root = Path(dirname(config_fn))
-    _cfdict = parse_relpath(cfdict.copy(), root)
+    _cfdict = make_config_paths_relative(cfdict.copy(), root)
     ext = splitext(config_fn)[-1].strip()
     if ext in [".yaml", ".yml"]:
-        _cfdict = _process_config_out(_cfdict)  # should not be done for ini
         with open(config_fn, "w") as f:
             yaml.dump(_cfdict, f, sort_keys=False)
-    elif ext == ".toml":  # user defined
-        _cfdict = _process_config_out(_cfdict)
-        with open(config_fn, "wb") as f:
-            dump_toml(_cfdict, f)
     else:
-        write_ini_config(config_fn, _cfdict, **kwargs)
-
-
-def _process_config_out(d):
```

Comment:
Why did you remove this method?

Reply:
because this was only necessary for toml support (since toml doesn't support `null` values, or a variant thereof) and I thought that was supposed to be removed. if `toml` stays then the function will stay too

Reply:
This method was also called for yaml, not only toml, that's why I asked. I also don't know (anymore) why the method is required in that case though. But I would suggest to keep it until we know its redundant.

Reply:
The reason it's there is that since TOML does not support some kind of `null` value, (the way you denote this in toml is simply by omitting the key). To make sure that we didn't lose any keys a special "NONE" value was inserted including in the YAML files, so when reading the YAML files back out agian these had to be removed as they were not necessary. 

git blame reveals that this was added in #444 (adding support for TOML files), so before it ran fine, and the tests pass without it so I'd say that's enough evidence for it  being redundant.